dev/com.ibm.ws.config/src/com/ibm/ws/config/xml/internal/SystemConfiguration.java

```diff
@@ -174,16 +188,17 @@ void start() throws ConfigUpdateException, ConfigValidationException, ConfigPars
             serverXMLConfig.loadInitialConfiguration(variableRegistry);
         }
 
-        final boolean reprocessConfig;
+        reprocessConfig();
+    }
+
+    void reprocessConfig() throws ConfigUpdateException {
         if (serverXMLConfig.isModified() || variableRegistry.variablesChanged()) {
             variableRegistry.clearVariableCache();
             changeHandler.updateAtStartup(serverXMLConfig.getConfiguration());
             serverXMLConfig.setConfigReadTime();
-            reprocessConfig = true;
-        } else {
-            reprocessConfig = false;
+            bundleProcessor.startProcessor(true);
         }
-        bundleProcessor.startProcessor(reprocessConfig);
+
```

Comment:
This is the change I did, but I think it is wrong.

Before when the config bundle started we could call BundleProcessor.startProcessor(false), but now we will always use `true` or we will not call `startProcessor` at all.  I'm pretty sure that is not what we want here.

Reply:
I passed a restoreHook flag to `SystemConfiguration.reprocessConfig(boolean)`